File: leetcode/119.pascals-triangle-ii.go
package leetcode

func getRow(rowIndex int) []int {
	arr := []int{1}
	for i := 0; i <= rowIndex; i++ {
		t := make([]int, i+1)
		t[0] = 1
		t[i] = 1
		for j := 1; j < i; j++ {
			t[j] = arr[j] + arr[j-1]
		}
		arr = t
	}
	return arr
}

File: leetcode/119.pascals-triangle-ii.ts
function getRow(rowIndex: number): number[] {
  const array = [1]
  for (let i = 1; i <= rowIndex; i++) {
    array[i] = 1
    let pre = 1
    for (let j = 1; j < i; j++) {
      const newN = array[j] + pre
      pre = array[j]
      array[j] = newN
    }
    // console.log(array)
  }
  return array
}
